inc0gnito-ctf-2015/.gitignore

```diff
@@ -0,0 +1,3 @@
+network/Hack-a-network-administrator's-terminal/network_hacking_102.zip
+forensic/POS/POS_9b648dbca2017349512c89466391c07d722418d1
+forensic/old_school/old_school_65c87efde678662c9139c81e016a62671cea2d6f
```

Comment:
It's too large. Here's links to files.
http://p.hyeon.me:8081/network_hacking_102.zip
http://p.hyeon.me:8081/POS_9b648dbca2017349512c89466391c07d722418d1
http://p.hyeon.me:8081/old_school_65c87efde678662c9139c81e016a62671cea2d6f

But don't directly link it, please. My server traffic limit is so tight.